component-interaction/src/app/user-list/user-list.component.ts

```diff
@@ -18,29 +18,42 @@ export class UserListComponent implements OnInit {
 	private prepareUsers(): void {
 		this.users = [
 			{
+				id: 1,
 				name: "Catalin",
 				email: "catalin@gmail.xyz"
 			},
 			{
+				id: 2,
 				name: "Mihai",
 				email: "mihai@gmail.xyz"
 			},
 			{
+				id: 3,
 				name: "Daniel",
 				email: "daniel@gmail.xyz"
 			},
 			{
+				id: 4,
 				name: "Iulia",
 				email: "iulia@gmail.xyz"
 			},
 			{
+				id: 5,
 				name: "popina-florin",
 				email: "popina-florin@gmail.xyz"
 			},
 			{
+				id: 6,
 				name: "gandore-mihai",
 				email: "gandore-mh@gmail.xyz"
 			}
 		]
 	}
+
+	public handleDeleteClick(event: number) {
+		const indexOfObject = this.users.findIndex(object => {
+			return object.id === event;
+		});
```

Comment:
```suggestion
		const indexOfUser = this.users.findIndex(
(user: UserModel) => user.id === userId
);
```

Reply:
Also you can write
this.users = this.users.filter(
    (user: UserModel) => user.id !== userId
);